install.sh

```
@@ -508,13 +508,37 @@ print_completion() {
     echo
 }
 
+setup_ssh_with_standalone_script() {
+    echo -e "${BLUE}Setting up SSH keys...${NC}"
+    echo
+    
+    local script_path="$(dirname "$0")/scripts/setup-ssh-keys.sh"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)